modules/shared/components/molecules/TrendeingQuestion/TrendingQuestion.tsx

```diff
@@ -1,37 +1,29 @@
 import React from 'react';
 import type { FC, ReactElement } from 'react';
-import Link from 'next/link';
 import ImagePoll from '../../icons/imagePoll.svg';
 import TextPoll from '../../icons/textPoll.svg';
 import MiniSurvey from '../../icons/miniSurvey.svg';
 import type { ITrendingQuestion } from './ITrendingQuestion';
 import styles from './TrendingQuestion.module.css';
-
-export const truncate = (str: string): string => {
-  const limit = 30;
-  return str.length > limit ? `${str.substr(0, limit - 1)}...` : str;
-};
+import UserInfo from '../UserInfo/UserInfo';
 
 const TrendingQuestion: FC<ITrendingQuestion.IProps> = ({
   postCaption,
   type,
 }): ReactElement => {
   return (
-    <div className="flex">
+    <UserInfo
+      isHidden={false}
+      variant="icon"
+      title={postCaption}
+      subTitle={type}
+    >
       <div className={styles['trending-question']}>
         {type === 'Mini Survey' && <MiniSurvey />}
         {type === 'Text Poll' && <TextPoll />}
         {type === 'Image Poll' && <ImagePoll />}
       </div>
-      <div className="w-60">
-        <div className="flex">
-          <Link href="/">
-            <a>{truncate(postCaption)}</a>
-          </Link>
-        </div>
-        <p className="text-xs text-dark-grey font-light">{type}</p>
-      </div>
-    </div>
+    </UserInfo>
```

Comment:
maybe UserInfo is not the best name for that component in this case. 
I think that ``TrendInfo`` or ``PostInfo`` would be better to be more readable in the future for anyone reading the code.
you can change the imported name or create new component called ``TrendInfo`` for example